PollApp.ts

```diff
@@ -20,7 +20,7 @@ import { createPollModal } from './src/lib/createPollModal';
 import { finishPollMessage } from './src/lib/finishPollMessage';
 import { votePoll } from './src/lib/votePoll';
 import { PollCommand } from './src/PollCommand';
-
+import { createMixedVisibilityModal } from './src/lib/createMixedVisibilityModal';
```

Comment:
There are a lot of tslint errors in this file. Eg on this line its throwing following error
`Import sources within a group must be alphabetized. (ordered-imports)`

So can you install some plugin in VS Code like `ESLint` so you can find and fix it directly from your editor.

Reply:
I've created this PR https://github.com/RocketChat/rocket.chat.app-poll/pull/9 where I've added new node script which you can also use for testing any lint issues locally